examples/configuration/README.md

```diff
@@ -12,28 +27,25 @@ dapr init
 npm install
 
 # Run the example directly using dapr run
-dapr run --app-id example-config --app-port 50051 --app-protocol http npm run start
+dapr run --app-id example-config --app-port 50051 --app-protocol grpc --resources-path ./components -- npm run start
 
 # or, using npm script
-npm run start:dapr-http
+npm run start:dapr-grpc
 ```
 
-## Switching from HTTP to gRPC
-
-By default, the example will run using HTTP. To use gRPC instead:
-
-- Add `CommunicationProtocolEnum.GRPC` to the DaprClient object creation:
-
-  ```typescript
-  const client = new DaprClient(daprHost, process.env.DAPR_HTTP_PORT, CommunicationProtocolEnum.GRPC);
-  ```
+## Sample output
 
-- To run:
-
-  ```bash
-  # Using dapr run
-  dapr run --app-id example-config --app-port 50051 --app-protocol grpc npm run start
-
-  # or, using npm script
-  npm run start:dapr-grpc
-  ```
+```
+== APP == {
```

Comment:
Should we remove the `== APP ==`?

Reply:
Actually we will get this log if we run it using the steps mentioned in the examples (because of Dapr CLI). I have removed these from docs because dapr and app can run separately too. I can remove it for clarity.

Reply:
Pushed, thanks! @XavierGeerinck